besu/src/main/java/org/hyperledger/besu/RunnerBuilder.java

```diff
@@ -190,6 +192,18 @@ public RunnerBuilder p2pEnabled(final boolean p2pEnabled) {
     return this;
   }
 
+  public RunnerBuilder p2pTLSConfiguration(final TLSConfiguration p2pTLSConfiguration) {
+    this.p2pTLSConfiguration = Optional.of(p2pTLSConfiguration);
+    return this;
+  }
+
+  public RunnerBuilder p2pTLSConfiguration(final Optional<TLSConfiguration> p2pTLSConfiguration) {
+    if (null != p2pTLSConfiguration) {
+      this.p2pTLSConfiguration = p2pTLSConfiguration;
+    }
+    return this;
+  }
+
```

Comment:
Are we using both these methods?

Reply:
yeah had to add them here to avoid conditional code from the callers :-)